### DISCUSSION TITLE
2024/10/11/macOS_installation/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#30] [General]
# 2024/10/11/macOS_installation/

多年前曾尝试过安装macOS虚拟机，因此这篇教程算复习。这两天帮助师兄做软件测试，需要macOS环境，由于缺乏mac设备，所以把整个虚拟机的安装流程又过了一遍。

https://wz.anoms.top/2024/10/11/macOS_installation/

**Comment by jingling000** [comment]
和哈哈哈哈哈哈哈哈哈我现在是macOS
